Online-cinema/js/menu.js

```diff
@@ -0,0 +1,28 @@
+const openMenu = (nav, activeClass) => {
+  nav.classList.add(activeClass);
+};
+
+const closeMenu = (nav, activeClass) => {
+  nav.classList.remove(activeClass);
+};
+
+export const slideMenu = ({
```

Comment:
I think it's a bad name. Would be better something like `initializeSlideMenu` or `createSlideMenu`